src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -73,6 +73,15 @@ public override ExportResult Export(in Batch<Activity> batch)
                     }
                 }
 
+                if (activity.Links.Any())
+                {
+                    this.WriteLine("Activity.Links:");
+                    foreach (var activityLink in activity.Links)
+                    {
+                        this.WriteLine($"    {activityLink.Context.TraceId} {activityLink.Context.SpanId}");
```

Comment:
Do we not want to print the `Tags` for the activity link?

Reply:
coming in future PRs.

Reply:
So far I haven't seen any semantic convention using Links much... The 1st one I am seeing is https://github.com/open-telemetry/opentelemetry-specification/pull/2078/files It doesn't yet use tags.